### ISSUE TITLE
:intro shows different version than other pages

### ISSUE THREAD

**Issue by scootergrisen** [vim/vim#3056]
In Vim 8.1 it says 8.1.1 on the :intro page.
The other pages like :version etc. says 8.1.

At least to me this is confusing.
I would expect to see the same version number on all pages.

**Comment by tonymec** [comment]
On my present Vim, `:intro` says `version 8.1.92` and the first two lines of `:version` are:
```
VIM - Vi IMproved 8.1 (2018 May 17, compiled Jun 21 2018 14:52:47)
Included patches: 1-92
```
The patchlevel (i.e. the third number of what `:intro` reports) is (when nonzero) at the end of `:version`'s second line. (When the patchlevel is zero, i.e. no additional patches since the minor release, that second line is omitted.) It has been so since as long as I knew Vim (6.1) and probably earlier. I see nothing contradictory or confusing in that. It's just that `version 8.1.92` is shorthand for:
```
Major version: 8
Minor version: 1
Patchlevel: 92
```

What concerns the help pages, which say `For Vim version 8.1`, the patchlevel is irrelevant for them because most of them don't change from one patchlevel to the next.

**Comment by scootergrisen** [comment]
If i go to https://www.vim.org/ and it says "Vim 8.1 is released!" i would expect it to say "version 8.1" when i start the program not 8.1.1.

**Comment by micbou** [comment]
You are just confused because your version is actually 8.1.1. Update to the latest version and you'll see 8.1.95 (as we speak).

**Comment by scootergrisen** [comment]
Why not say the same version throughout the program?

**Comment by tonymec** [comment]
Version 8.1 refers to all patchlevels from 8.1.0 to the current 8.1.95 and beyond. Similarly version 8.0, which isn't the latest one anymore, refers to all patchlevels from 8.0.0 to 8.0.1850. So it is the same version throughout, except that `:intro` gives you a more detailed info (the patchlevel) and `:version` is still more detailed (on its second line, when present, it gives you the list of all included patches in this build of this major.minor version).

"Version 8" could even be used to mean all versions and patchlevels from 8.0.0 (included) to some future 9.0.0 (excluded). For instance the helpfile `version7.txt` gives info about everything new and changed in version 7, from the changes in 7.0.0 relative to version 6 up to the 22 patchlevels of 7.4b beta. For some reason patchlevels 7.4.1 to 7.4.2367 are not detailed there (yet?), but IMHO they ought to be.

Best regards,
Tony.

**Comment by k-takata** [comment]
> For some reason patchlevels 7.4.1 to 7.4.2367 are not detailed there (yet?), but IMHO they ought to be.

They are listed in version8.txt. (I feel it is confusing.)

(I fixed a typo in the title.)
